src/pages/Browse/index.jsx

```diff
@@ -265,8 +267,16 @@ const Browse = () => {
                 {browseDataList?.map((data) => (
                   <tr key={data.id} className="text-center">
                     <td>
-                      <div className="font-demi">{data?.uploadname}</div>
-                      <div className="font-size-small">{data?.description}</div>
+                      <Link
+                        to={`${routes.browseUploads.licenseBrowser}?uploadID=${data.id}`}
```

Comment:
Can upload ID be part of URL instead of get parameter like in the `uploads/{id}/licenses` endpoint? It will look nice.

Reply:
@GMishx I have added the query parameters because it is currently implemeted in the fossology code base but now I have updated the route with `browse/licenseBrowser/:uploadId`

Reply:
There is no correct way to do it actually just what is easier to do. Adding the ID to path also enables browser to cache the content much easily and make it easy for user to remember/understand the path.
Speaking of it `browse/:uploadId/licenseBrowser` will be an even better path. Doing so, if user wants to check the copyrights, only the last portion of the path will change (`browse/:uploadId/copyright`). This allows to create upload ID as an abstract entity in the server and everything following it in the path becomes a property of it (like object variables of a class).

Do you think making the change will be possible and easier to implement?

Reply:
We can do that but there is a problem in it for currently active tab we are using route name https://github.com/fossology/FOSSologyUI/pull/146/files#diff-97a95676f87f073a40ceb077030b0f84df7f999e10e59c02d3e98bd6dc1ad76bR56-R58 and currently i just need to check whether it includes that variable because in variable i can't declare it line `browse/:uploadId/licenseBrowser` otherwise it will consider whole path as constant rather than taking `:uploadId` as dynamic and for showing the active tab we need to declare different variable for same paths that might not be a good idea.

Reply:
Hmmm... Okay. It is fine for initial release but it will really look odd going further.

Reply:
There is a conflict. Please rebase

Reply:
@GMishx rebased the pr